src/platform/src/core/utils/proxyClient.js

```
@@ -87,26 +98,88 @@ export const createProxyHandler = (options = {}) => {
+      // Create request config with enhanced error handling
+      let API_BASE_URL;
+      try {
+        API_BASE_URL = getApiBaseUrl();
+      } catch (envError) {
+        logger.error('Failed to get API base URL from environment:', envError);
+        const errorResponse = {
+          success: false,
+          message: 'API configuration error: Unable to determine base URL',
+        };
+
+        if (context && context.params) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)